Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.14.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
I generally prefer to use `~> <MAJOR>.<MINOR>` to ensure bug fixes and minor platform support updates are included but in the next sprint or two I am hoping to convert all teams away from `sqa-utils`/`genconfig2` to the new `beaker-hostgenerator` gem.